_automating-configurations/workflow-steps.md

```diff
@@ -75,9 +75,13 @@ You can include the following additional fields in the `user_inputs` field if th
 
 You can include the following additional fields in the `previous_node_inputs` field when indicated.
 
-|Field	|Data type	|Description	|
-|---	|---	|---	|
-|`model_id`	|String	|The `model_id` is used as an input for several steps. As a special case for the Register Agent step type, if an `llm.model_id` field is not present in the `user_inputs` and not present in `previous_node_inputs`, the `model_id` field from the previous node may be used as a backup for the model ID.	|
+| Field	          |Data type	| Description	                                                                                                                                                                                                                                                                                                                                                                               |
+|-----------------|---	|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `model_id`	     |String	| The `model_id` is used as an input for `delete_model`, `deploy_model`, `undeploy_model`, `register_agent` and `create_tool` step. As a special case for the `register_agent` step type, if an `llm.model_id` field is not present in the `user_inputs` and not present in `previous_node_inputs`, the `model_id` field from the previous node may be used as a backup for the model ID.	 |
```

Comment:
This is still missing the fact that for create_tool, it's part of the parameters input.

That's really the essential information you're adding with this documentation update but it's not obvious to me and I've looked at the code. 

I really don't think you need to list every step that it's a required input for; those are included in the API already.  The previous line "several steps" is fine, perhaps adding a reference to using that workflow step API.

Please focus the new content on what is _not_ clearly identified by the existing API.

Reply:
I see.